### DISCUSSION TITLE
ASUS System Control Interface can't be installed

### DISCUSSION THREAD

**Discussion by ismoldayev** [seerge/g-helper#1291] [General]
When I try to open G-Helper I get the error message "Can't connect to Asus ACPI". When I try to install Asus System Control Interface I am met with "The version of Asus System Control driver of your machine cannot support some function." When I try to open G-Helper again I get the same error as before.

I am using a Asus TUF B650 PLUS WIFI

**Comment by seerge** [comment]
It's a tool for laptops, not desktops

<img width="869" alt="Screenshot 2023-09-11 at 00 13 14" src="https://github.com/seerge/g-helper/assets/5920850/3ef9fe66-fc0c-4420-a9c1-b6c23c33da44">
